packages/i18n/locales/en.json

```
@@ -168,7 +168,8 @@
       "overview": "Overview",
       "minorVersions": "Minor versions",
       "releaseAnnouncement": "Release Announcement",
-      "unsupportedVersionWarning": "This version is out of maintenance. Please use a currently supported version."
+      "unsupportedVersionWarning": "This version is out of maintenance. Please use a currently supported version. <link>Understand EOL support.</link>",
+      "ltsVersionFeaturesNotice": "Want new features sooner? Get the <link>latest Node.js version</link> instead and try the latest improvements on Node!"
```

IIRC "Node" shouldn't be used to describe the project.

```suggestion
      "ltsVersionFeaturesNotice": "Want new features sooner? Get the <link>latest Node.js version</link> instead and try the latest improvements!"
```

---

It's a forbidden term (lint-wise) in core

---

On core, yes, not here. If we want it to be forbidden, open a PR 🙇

---

The logic is kinda backward here. It's not that "the linter forbids to use `Node` on core so we may not use it there", it's "the project decided it was important to be consistent when naming the project and so the lint rule was added". The lint rule helps enforce the rule, but is not the rule, and the rule still exist without the linter enforcing it – it's up to us human reviewers to enforce it.